src/ui_interface.h

```diff
@@ -122,7 +122,6 @@ void InitWarning(const bilingual_str& str);
 
 /** Show error message **/
 bool InitError(const bilingual_str& str);
-constexpr auto AbortError = InitError;
 
```

Comment:
```suggestion
inline bool AbortError(const bilingual_str& str) { return InitError(str); }

```

Reply:
Which is more readable too.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19309#issuecomment-645582741).